pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -3,8 +3,21 @@ title: Included in the accessibility tree
 key: included-in-the-accessibility-tree
 ---
 
-Elements included in the accessibility tree of platform specific accessibility APIs. Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+Elements included in the accessibility tree of platform specific accessibility APIs. 
 
-The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are [included in the accessibility tree][] can be found in the [HTML accessibility API mappings](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings](https://www.w3.org/TR/svg-aam/).
+Elements in the accessibility tree are exposed to assistive technologies, allowing users to interact with the elements in a way that meet the requirements of the individual user.
+
+An element is included in the accessibility tree if it is a DOM node in the flattened DOM tree that meets the following criteria:
+
+1. It isn't hidden through WAI-ARIA or CSS, and
+2. it has one or more of the following:
+   * It has a [semantic role](#semantic-role), or
+   * It has an accessible name that, when trimmed of whitespace, is not an empty string, or
+   * it owns another element though an `aria-owns` attribute, or
+   * It is owned by another element that references it with `aria-owns`
+
+ > **Note:** Text nodes and pseudo elements can also be part of the accessibility tree, since they have their text content as the accessible name.
+
+The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are [included in the accessibility tree](#included-in-the-accessibility-tree) can be found in the [HTML accessibility API mappings](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings](https://www.w3.org/TR/svg-aam/).
```

Comment:
I'm not sure this paragraph needs to be here. These mapping documents describe how to map attributes to the accessibility APIs. It doesn't say much about which elements are and which aren't part of the accessibility tree.

Reply:
@WilcoFiers, I just looked into it, and it seems that the core-aam has some pretty neat guidance on when things should [excluded](https://www.w3.org/TR/core-aam/#exclude_elements2) from and [included](https://www.w3.org/TR/core-aam/#include_elements) in the accessibility tree.
Reading this, we can see things that we have gotten wrong so far, e.g. missing the html5 `hidden` attribute as a way of excluding things from the accessibility tree, and that focusable elements per the spec should be included in the accessibility tree...

Will check out the other two specs for relevance later...

Reply:
@WilcoFiers At least SVG AAM defines additional criteria for when elements are included in/excluded from the accessibility tree, see https://www.w3.org/TR/svg-aam/#include_elements and https://www.w3.org/TR/svg-aam/#exclude_elements. As such, SVG elements have different requirements than e.g. HTML for when accessible objects are created in the accessibility tree and do not fit the criteria currently listed.